src/util/index.js

```diff
@@ -1,25 +1,75 @@
 import axios from "axios";
- 
+
+// ``** GET **``
+
+// GET params example
+const params = {
+  params: {
+    ID: 12345,
+  },
+};
+
 // note: not used, but could be used with GET with params
- const getData = async (url, params) => {
-    try {
-      let res = await axios.get(url, params);
-      let data = await res.data;
-      return data;
-    } catch (error) {
-      console.log(error, `error - getData in ${url} route`);
-    }
-  };
-
-  const getAllData = async (url) => {
-    try {
-      let res = await axios.get(url);
-      let data = await res.data;
-      return data;
-    } catch (error) {
-      console.log(error, `error - getAllData in ${url} route`);
-    }
-  };
-
-
-  export {getData, getAllData};
\ No newline at end of file
+const getData = async (url, params) => {
+  try {
+    let res = await axios.get(url, params);
+    let data = await res.data;
```

Comment:
here we don't need to use await again, since we already used await once, .data is already a property and not a promise

Reply:
The pattern where we await the line immediately after a fetch is when we use .json, but since axios is sending, receiving and then parsing the response for us, there is only one asynchronous function that needs the await key word.